src/System.AppContext/src/project.json

```diff
@@ -1,21 +1,24 @@
 {
   "frameworks": {
-    "netstandard1.6": {
+    "netstandard1.7": {
       "dependencies": {
         "Microsoft.TargetingPack.Private.CoreCLR": "1.1.0-beta-24512-03"
       },
       "imports": [
-        "dotnet5.6"
```

Comment:
This is the wrong mapping. It isn't actually needed so you can remove all the imports in this file.